crazy-filters/transforms.py

```diff
@@ -647,11 +647,13 @@ def face_overlay(img, overlay, relative_position=0.2, relative_width=1.):
     for x, y, w, h in faces:
         # pour chaque visage, colle l'image un peu au-dessus du rectangle
         # tu peux afficher le rectangle en décommentant la ligne suivante
-        # cv2.rectangle(img, (x, y), (x + w, y + h), (255, 0, 0), 2)
-        new_y = max(0, int(y + relative_position * h))
-        new_width = int(w * relative_width)
-        new_x = x - (new_width - w) // 2
-        paste_image(img, overlay, new_x, new_y, fit_width=new_width, make_transparent=(255, 255, 255))
+        if overlay is None:

```

Comment:
I'm partial to "return early" so I'd advise having a `continue` just after the cv2.rectangle(...)